internal/storage/sqlite/backup.go

```
@@ -168,6 +168,22 @@ func validateRequiredSchema(ctx context.Context, db *sql.DB, version int) error
 			return fmt.Errorf("close required schema probe: %w", err)
 		}
 	}
+	if version >= 25 {
+		rows, err := db.QueryContext(ctx, "SELECT 1 FROM bus_vehicle_positions LIMIT 0")
+		if err != nil {
+			return fmt.Errorf("required real-time vehicle position schema is unavailable")
+		}
+		if err := rows.Close(); err != nil {
+			return fmt.Errorf("close real-time schema probe: %w", err)
+		}
+		rows, err = db.QueryContext(ctx, "SELECT realtime_authorized FROM bus_source_revisions LIMIT 0")
```

golangci-lint (2.12.2)

[error] rows.Err must be checked

(rowserrcheck)